app/views/curation_concerns/generic_files/_actions.html.erb

```diff
@@ -1,16 +1,16 @@
-<% if can?(:edit, generic_file) %>
+<% if can?(:edit, generic_file.id) %>
```

Comment:
We don't have any abilities defined for `can :edit, GenericFilePresenter`, but we do have an ability for `can :edit, "my-object-id"`